gs.sh

```diff
@@ -1,22 +1,14 @@
-echo "\n$(tput setaf 5)consumer-layouts status:$(tput sgr0)"
-cd consumer-layouts
-git status
-
-echo "\n$(tput setaf 5)consumer-domain status:$(tput sgr0)"
-cd ../consumer-domain
-git status
-
-echo "\n$(tput setaf 5)consumer-session status:$(tput sgr0)"
-cd ../consumer-session
-git status
-
-echo "\n$(tput setaf 5)domain-values status:$(tput sgr0)"
-cd ../domain-values
-git status
-
-echo "\n$(tput setaf 5)consumer-main status:$(tput sgr0)"
-cd ../consumer-main
-git status
-
-cd ..
+#!/usr/bin/env bash
+
+DIRS="consumer-layouts
+consumer-domain
+consumer-session
+domain-values
+consumer-main"
+
+for dir in $DIRS
+do
+  echo "$(tput setaf 5)$dir status:$(tput sgr0)"
+  git --git-dir=$dir/.git --work-tree=$dir status
```

Comment:
Mind giving me a quick explanation of this, Aaron? I want to learn :)
Also, thanks for making this half its original size!!